docs/README.md

```diff
@@ -73,6 +73,7 @@
   * [Debugging with Chrome Developer Tools](articles/documentation/recipes/debugging-with-chrome-dev-tools.md)
   * [Debugging with Visual Studio Code](articles/documentation/recipes/debugging-with-visual-studio-code.md)
   * [Finding Code Issues with Flow Type Checker](articles/documentation/recipes/finding-code-issues-with-flow-type-checker.md)
+  * [Integrating TestCafe with TeamCity CI System](articles/documentation/recipes/integrating-testcafe-with-team-city-ci-system.md)
```

Comment:
May be we should add special sections at least for 'Integration with CI Systems'?

Reply:
Yep, I'm definitely going to do this